lib/minitest/snapshots.rb

```diff
@@ -1,19 +1,3 @@
 require "minitest"
 
-module Minitest::Snapshots
```

Comment:
🗒️ I moved this code to `snapshots_plugin.rb` (i.e. the common entry point for Minitest v5 and v6).